<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TM3</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../TM3res.qrc">
    <normaloff>:/icons.icns</normaloff>:/icons.icns</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>48</width>
    <height>48</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>40</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>41</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QToolButton" name="btnAdd">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnUpdate">
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnDel">
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="btnReport">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>200</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="btnDateBack">
               <property name="toolTip">
                <string>1 day backward</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../TM3res.qrc">
                 <normaloff>:/images/new/left-pointing-arrow.png</normaloff>:/images/new/left-pointing-arrow.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit">
               <property name="font">
                <font>
                 <family>Tahoma</family>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnToday">
               <property name="toolTip">
                <string>Today</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../TM3res.qrc">
                 <normaloff>:/images/new/homepage.png</normaloff>:/images/new/homepage.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnDateFwd">
               <property name="toolTip">
                <string>1 day forward</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../TM3res.qrc">
                 <normaloff>:/images/new/right-arrow.png</normaloff>:/images/new/right-arrow.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblLoad">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>171</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>140</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Tahoma</family>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>437</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableActs">
         <property name="font">
          <font>
           <family>Tahoma</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menuRB">
    <property name="title">
     <string>Refbooks</string>
    </property>
    <addaction name="actNew2"/>
    <addaction name="actUnits"/>
    <addaction name="actRates"/>
    <addaction name="actTags"/>
    <addaction name="actCat"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRB"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit to desktop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionActNew">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/new/check-mark.png</normaloff>:/images/new/check-mark.png</iconset>
   </property>
   <property name="text">
    <string>Add action</string>
   </property>
   <property name="toolTip">
    <string>New action</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actToggleToolBar">
   <property name="text">
    <string>Hide toolbar</string>
   </property>
   <property name="toolTip">
    <string>Hide toolbar</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actUnits">
   <property name="text">
    <string>Units</string>
   </property>
  </action>
  <action name="actRates">
   <property name="text">
    <string>Rates</string>
   </property>
  </action>
  <action name="actCat">
   <property name="text">
    <string>Cats</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/new/pencil.png</normaloff>:/images/new/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Update action</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/new/cross.png</normaloff>:/images/new/cross.png</iconset>
   </property>
   <property name="text">
    <string>Delete action</string>
   </property>
   <property name="toolTip">
    <string>Delete action</string>
   </property>
   <property name="shortcut">
    <string>Alt+Del</string>
   </property>
  </action>
  <action name="actionSimpleReport">
   <property name="icon">
    <iconset resource="../TM3res.qrc">
     <normaloff>:/images/new/edit-document.png</normaloff>:/images/new/edit-document.png</iconset>
   </property>
   <property name="text">
    <string>Time report</string>
   </property>
   <property name="toolTip">
    <string>Time report</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actTags">
   <property name="text">
    <string>Tags</string>
   </property>
  </action>
  <action name="actNew2">
   <property name="icon">
    <iconset>
     <normaloff>:/images/list-add.png</normaloff>:/images/list-add.png</iconset>
   </property>
   <property name="text">
    <string>Add action</string>
   </property>
   <property name="toolTip">
    <string>New Action</string>
   </property>
   <property name="shortcut">
    <string>Ins</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../TM3res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionActNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addAct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actUnits</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actUnitsDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actRates</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actRates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCat</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actCatDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableActs</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateAct(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>579</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDel</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>delAct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>updateAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>MainWindow</receiver>
   <slot>dateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDateBack</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dayBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>-51</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDateFwd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>dayFwd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSimpleReport</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actSimpleReport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnToday</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>today()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>-62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actTags</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actTagDlg()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actNew2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>addAct2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>about()</slot>
  <slot>addAct()</slot>
  <slot>toggleToolBar()</slot>
  <slot>actUnitsDlg()</slot>
  <slot>actRates()</slot>
  <slot>actCatDlg()</slot>
  <slot>updateAct(QModelIndex)</slot>
  <slot>delAct()</slot>
  <slot>dayBack()</slot>
  <slot>dayFwd()</slot>
  <slot>showCalendar()</slot>
  <slot>updateAction()</slot>
  <slot>dateChanged()</slot>
  <slot>actSimpleReport()</slot>
  <slot>today()</slot>
  <slot>actTagDlg()</slot>
  <slot>actBudjetDlg()</slot>
  <slot>actDaysDlg()</slot>
  <slot>addAct2()</slot>
 </slots>
</ui>
